src/sections/Events/index.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { MeetupStyle } from "./meetups.style";
+import Card from "../../components/Card"
+import {Col, Container, Row} from "../../reusecore/Layout";
+import PageHeader from "../../reusecore/PageHeader";
+
+const Meetup = (data) =>{
+    return(
+        <MeetupStyle>
+            <PageHeader title="Events" path="Community > Events"/>
+            <h2>Meetups</h2>
+            <div>
+                <Container>
+                    <div>
+                        <Row>
+                            {data.data.meetups.nodes.map(({id,frontmatter, fields }) => (
+                                <Col xs={12} sm={6} lg={6} key={id}>
+                                    <Card frontmatter={frontmatter} fields={fields}/>
```

Comment:
You can enclose this Card tag inside an anchor tag with src as the eurl to override the Link generated by fields.slug.